src/NUnitEngine/nunit.engine/Guard.cs

```diff
@@ -23,7 +23,7 @@
 
 using System;
 
-namespace NUnit.Engine
+namespace NUnit.Common
```

Comment:
We might want to come back and gather the various common classes into a separate directory somewhere.